tkweb/apps/drinks/admin.py

```diff
@@ -0,0 +1,22 @@
+from django.contrib import admin
+
+from .models import Drink, Sprut, Barcard 
```

Comment:
Generelt bruger vi absolutte imports i tkweb:

```suggestion
from tkweb.apps.drinks.models import Drink, Sprut, Barcard
```